discord/flags.py

```diff
@@ -1436,3 +1465,110 @@ class ChannelFlags(BaseFlags):
     def pinned(self):
         """:class:`bool`: Returns ``True`` if the thread is pinned to the forum channel."""
         return 1 << 1
+
+
+class ArrayFlags(BaseFlags):
```

Comment:
I'm not really big on the changes done to flags.py, sorry. Please keep `.value` as `.value` and do not hide it under `_value`. 

You can keep ArrayFlags mostly as-is but just add a method for the actual AutoModPresets class to transform it into an actual list instead of modifying everything to fit the change.